pymc/tests/test_idata_conversion.py

```diff
@@ -637,6 +636,14 @@ def test_issue_5043_autoconvert_coord_values(self):
             )
             assert isinstance(converter.coords["city"], pd.MultiIndex)
 
+    def test_elemwise_log_likelihood(self):
```

Comment:
I would include the check on log_likelihood shape into _multiple_ other tests so we check multiple scenarios like multiple observed variables, multivariate observed variables...

`test_to_idata` for example uses the `check_multiple_attrs` helper so it checks for presence or non-presence of keys as groups and of values as variables. But it then has an extra test on posterior predictive to make sure the reshaping is working correctly. We could add the test for log likelihood shape here to cover the same scenario as this new test.

Then it would probably be good to add checks in `test_missing_data_model`, `test_mv_missing_data_model`, `test_multiple_observed_rv` and `test_multivariate_observations`.

Reply:
Yeah I can do that, then we don't even need this test

Reply:
I added a log_likelihood shape checks to pre-existing tests